starter/class/cursor.js

```diff
@@ -2,17 +2,27 @@ const Screen = require("./screen");
 
 class Cursor {
 
-  constructor(numRows, numCols) {
+  constructor(numRows, numCols, startRow, startCol) {
```

Comment:
if you dont specify the start row/col, then you can do

```js
 constructor(numRows, numCols, startRow = 0, startCol = 0)
```

Reply:
would also recommend you set the same defaults for numRows and numCols

Reply:
I ended up not using these parameters, I'll remove them on my next push.